python/apigw-lambda-rds/README.md

```diff
@@ -0,0 +1,44 @@
+# Api Gateway with Lambda & RDS (Mysql)
+
+This example creates an API Gateway with a Lambda that will query data from the RDS Mysql Instance. 
+
+After the RDS Instance is provisioned a Cloudformation custom resource is used to create a DB populate with some example data.
+
+The lambda function & custom resource use Secrets Manager for storage & retrieval of DB connection details.
+
+## Prep
+
```

Comment:
can you also include the commands to get the virtual env up ahead of the install commands